configs/sst_kernel_tps-intel-qat.yaml

```diff
@@ -5,52 +5,53 @@ data:
   description: Userspace libraries and tools for Intel QAT hardware crypto device
   maintainer: sst_kernel_tps
 
-# packages:
-#  - do not exist in fedora yet
+  packages:
```

Comment:
This part is unnecessary.  It'll list these packages for architectures where that package doesn't exist.

Reply:
removed the packages: part. qatlib is now listed in the arch_packages:/x86_64: section only.
thanks josh!

Reply:
i hope the yaml is now correct. apologies for the mess.